### PR TITLE
Fix groupversion for multicasskop

### PR DESCRIPTION
**Pull request by AKamyshnikova**
Also updated setup namespace for casskop(drop deprecated usage of namespace)

Fixes: #123

| Q               | A
| --------------- | ---
| Bug fix?        | [x]
| New feature?    | []
| API breaks?     | []
| Deprecations?   | []
| Related tickets | fixes #X, partially #Y, mentioned in #Z
| License         | Apache 2.0

### CODE REVIEW

**Comment by cscetbon** [comment]
Were you able to test it locally @AKamyshnikova ?

**Comment by AKamyshnikova** [comment]
@cscetbon Yes, checked locally

`root@ataraday-3:~# helm upgrade -i multi-casskop oci://ghcr.io/cscetbon/multi-casskop-helm --set k8s.local=k8s-cluster1 --set-json k8s.remote={} -n cassandra --create-namespace --set image.tag=multi-casskop
 --set image.repository=docker.io/ataraday/cassandra-op                                                
Release "multi-casskop" does not exist. Installing it now.                                             
Pulled: ghcr.io/cscetbon/multi-casskop-helm:2.1.19
Digest: sha256:c1ae37d3063556d907708730059282319af722bead82bd399b3896cc843e19d0                        
W1026 07:30:18.995320 3733769 warnings.go:70] unknown field "spec.subresources"                        
coalesce.go:223: warning: destination for multi-casskop-helm.k8s.remote is a table. Ignoring non-table value ([k8s-cluster2])                                                                                 
NAME: multi-casskop                                                                                    
LAST DEPLOYED: Thu Oct 26 07:30:19 2023                                                                
NAMESPACE: cassandra                                                                                   
STATUS: deployed                                                                                       
REVISION: 1                                                                                            
TEST SUITE: None                                                                                                                                                                                              
NOTES:                                                                                                 
Congratulations. You have just deployed Multi-CassKop Operator.                                                                                                                                               
Check its status by running:                                                                                                                                                                                  
kubectl --namespace cassandra get pods -l "release=multi-casskop"                                                                                                                                             
                                                                                                                                                                                                              
Visit https://github.com/cscetbon/casskop/multi-casskop for instructions on how to create & configure Cassandra clusters using the operator.                                                                  
root@ataraday-3:~# kubectl -n cassandra get pods                                                                                                                                                              
NAME                                                READY   STATUS    RESTARTS   AGE                                                                                                                          
multi-casskop-multi-casskop-helm-56b6f999d5-x2xx4   0/1     Running   0          5s 
root@ataraday-3:~# kubectl -n cassandra logs -f multi-casskop-multi-casskop-helm-56b6f999d5-x2xx4
time="2023-10-26T07:30:41Z" level=info msg="Go Version: go1.19.13"
time="2023-10-26T07:30:41Z" level=info msg="Go OS/Arch: linux/amd64"
time="2023-10-26T07:30:41Z" level=info msg="multi-casskop Version: "
time="2023-10-26T07:30:41Z" level=info msg="multi-casskop Compilation Date: "
time="2023-10-26T07:30:41Z" level=info msg="Configuring Client for local cluster k8s-cluster1. using local k8s api access"
time="2023-10-26T07:30:41Z" level=info msg="Creating Controller"
time="2023-10-26T07:30:41Z" level=info msg="Create Client for local Cluster k8s-cluster1"
time="2023-10-26T07:30:42Z" level=info msg="Add CRDs to Cluster k8s-cluster1 Scheme"
time="2023-10-26T07:30:42Z" level=info msg="Configuring Watch for MultiCasskop on local"
time="2023-10-26T07:30:42Z" level=info msg="Starting Manager."`

File: charts/multi-casskop/templates/deployment.yaml [hunk 1]
```diff
@@ -50,20 +50,6 @@ spec:
 {{- range .Values.k8s.remote }}
           - -r{{ . }}
 {{- end }}
-        readinessProbe:
```
Comment (cscetbon) [inline charts/multi-casskop/templates/deployment.yaml:53]: Can you elaborate on this ? This is managed by k8s itself so why remove this ?

Comment (AKamyshnikova) [reply charts/multi-casskop/templates/deployment.yaml:53]: These are health checks provided with controller-runtime - in casskop we have https://github.com/cscetbon/casskop/blob/master/main.go#L205
But multicluster-controller does not have such functionality.

**Review by cscetbon — APPROVED** [review]

**Pull request merged**